app/components/Header/DropdownContainer/FadeContents.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+
+import React, { forwardRef } from 'react';
```

Comment:
Remember to remove the React import from all files. We don't need them anymore

Reply:
Good point!